actors/market/src/state.rs

```diff
@@ -85,6 +88,7 @@ pub struct State {
 
 /// IDs of deals associated with a single sector.
 #[derive(Clone, Debug, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
+#[serde(transparent)]
```

Comment:
This would need a FIP change wouldn't it? Currently it's like this in FIP-0076:

```
struct SectorDeals {
    Deals: []DealID
}
```

But this change would effectively make it something like: `SectorDeals []DealID`

Reply:
https://github.com/filecoin-project/go-state-types/pull/236 makes this assumption too, we go from `[[deal1,deal2,deal3,...]]` to `[deal1,deal2,deal3,...]`, which seems reasonable (although only saving 1 or 2 bytes), but the format seems to be assumed by a plain reading of 0076 with tuple structs.

Reply:
Thanks, I'll fix them too.

Reply:
See https://github.com/filecoin-project/FIPs/pull/917 and https://github.com/filecoin-project/go-state-types/pull/238

Reply:
NOTE: we can go either way. The only reason I bring this up is that, e.g., the FIP states that `SectorContentChangedParams` should be transparent (so I figured we might as well be consistent).

Reply:
Yes, the reason it wasn't already is that an earlier version had another field in here.